ct.yaml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+chart-dirs:
+  - charts
+chart-repos:
+  - bitnami=https://charts.bitnami.com/bitnami
```

Comment:
I believe this can be removed as we do not need dependency charts from bitnami

Reply:
Thanks for catching it. Submitted another PR https://github.com/open-telemetry/opentelemetry-helm-charts/pull/9